ecommerce/templates/payment/lumsxpay.html

```diff
@@ -0,0 +1,123 @@
+{% extends "edx/base.html" %}
+{% load i18n %}
+{% block title %}Bootstrap Demo{% endblock title %}
+
+
+{% block content %}
+    <nav class="navbar navbar-default navbar-fixed-top">
+        <div class="container-fluid">
+            <a class="navbar-brand" rel="home" href="#" title="Buy Sell Rent Everyting">
+                <img style="max-width:160px; margin-top: -20px; margin-left: 15px;"
+                     src="{{ configuration_helpers.BRANDING.logo }}">
+            </a>
+            <div class="collapse navbar-collapse" id="main-navbar-collapse">
+                <ul class="nav navbar-nav navbar-right">
+                    {% if user.is_authenticated %}
+                        <li class="btn-group user-menu" style="margin-right: 50px">
+                            <button type="button" class="btn btn-default hidden-xs main-btn nav-button"
+                                    onclick="window.open('{{ lms_dashboard_url }}');">
+                                <i class="icon fa fa-home" aria-hidden="true"></i>
+                                <span class="sr-only">{% trans "Dashboard for:" %}</span>
+                                {{ user.username }}
+                            </button>
+                        </li>
+                    {% else %}
+                        <a class="btn btn-primary navbar-btn hidden-xs" href="{% url 'login' %}">{% trans "Login" %}</a>
+                        <li class="visible-xs"><a class="nav-link" href="{% url 'login' %}">{% trans "Login" %}</a></li>
+                        </a>
+                    {% endif %}
+                </ul>
+            </div>
+        </div>
+
+    </nav>
+    <div class="container">
+        <div class="alert alert-warning" style="max-width: none;">
+            Payment clearance to the course might take upto 45 minutes.
```

Comment:
shouldn't minutes in msg be dynamic, if we ever change the frequency of cron job

Reply:
yes we can do that